src/wiki/keyboard-shortcuts.md

```diff
@@ -0,0 +1,208 @@
+| <kbd>Win</kbd> + <kbd>↓</kbd>                                           | Minimizes the desktop window.                                                                                                           |
+| <kbd>Win</kbd> + <kbd>←</kbd>                                           | Maximizes the desktop window to the left side of the screen.                                                                            |
+| <kbd>Win</kbd> + <kbd>→</kbd>                                           | Maximizes the desktop window to the right side of the screen.                                                                           |
+| <kbd>Win</kbd> + <kbd>Home</kbd>                                        | Minimizes all except the active desktop window. (Restores all windows on second stroke.)                                                |
+| <kbd>Win</kbd> + <kbd>Shift</kbd> + <kbd>↑</kbd>                        | Stretches the desktop window to the top and bottom of the screen.                                                                       |
+| <kbd>Win</kbd> + <kbd>Shift</kbd> + <kbd>↓</kbd>                        | Restores the active window vertically, maintaining width.                                                                               |
+| <kbd>Win</kbd> + <kbd>Shift</kbd> + <kbd>→</kbd> or <kbd>←</kbd>        | Moves the desktop window from one monitor to another.                                                                                   |
+| <kbd>Win</kbd> + <kbd>Spacebar</kbd>                                    | Switches input languages and keyboard layouts.                                                                                          |
+| <kbd>Win</kbd> + <kbd>Ctrl</kbd> + <kbd>Spacebar</kbd>                  | Changes to a previously selected input.                                                                                                 |
+| <kbd>Win</kbd> + <kbd>Enter</kbd>                                       | Opens Narrator.                                                                                                                         |
+| <kbd>Win</kbd> + <kbd>/</kbd>                                           | Launches the IME (Input Method Editors) reconversion.                                                                                   |
+| <kbd>Win</kbd> + <kbd>+</kbd> or <kbd>-</kbd>                           | Zooms in or out using Magnifier.                                                                                                        |
+| <kbd>Win</kbd> + <kbd>Esc</kbd>                                         | Stops Magnifier.                                                                                                                        |
+
+# File Explorer keyboard shortcuts
```

Comment:
```suggestion
## File Explorer keyboard shortcuts
```

For better accessibility, only use H1s for the article title, and to make the "On this page" functionality working properly.